api/lib/domain/usecases/dismiss-live-alert.js

```diff
@@ -0,0 +1,19 @@
+import { NotFoundError } from '../errors.js';
+
+export const dismissLiveAlert = async ({ userId, sessionId, certificationChallengeLiveAlertRepository }) => {
+  const certificationChallengeLiveAlert =
+    await certificationChallengeLiveAlertRepository.getOngoingBySessionIdAndUserId({
+      sessionId,
+      userId,
+    });
+
+  if (!certificationChallengeLiveAlert) {
+    throw new NotFoundError();
```

Comment:
Je propose de mettre un message, ça sera cool d'avoir un truc explicite dans les logs si jamais ça se produit.

```suggestion
    throw new NotFoundError("The challenge is not currently in alert");
```